convert_jendl.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+
+import argparse
+import glob
+import os
+import sys
+import tarfile
+import zipfile
+from collections import defaultdict
+from string import digits
+from urllib.request import urlopen
+
+import openmc.data
+from openmc._utils import download
+
+description = """
+Download JENDL 4.0 data from JAEA and convert it to a HDF5 library for
+use with OpenMC.
+
+"""
+
+
+class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
+                      argparse.RawDescriptionHelpFormatter):
+    pass
+
+
+parser = argparse.ArgumentParser(
+    description=description,
+    formatter_class=CustomFormatter
+)
+parser.add_argument('-b', '--batch', action='store_true',
+                    help='supresses standard in')
+parser.add_argument('-d', '--destination', default=None,
+                    help='Directory to create new library in')
+parser.add_argument('--libver', choices=['earliest', 'latest'],
+                    default='latest', help="Output HDF5 versioning. Use "
+                    "'earliest' for backwards compatibility or 'latest' for "
+                    "performance")
+parser.add_argument('-r', '--release', choices=['4.0'],
+                    default='4.0', help="The nuclear data library release version. "
+                    "The only option currently supported is 4.0")
+args = parser.parse_args()
+
+
+
+library_name = 'jendl' #this could be added as an argument to allow different libraries to be downloaded
+endf_files_dir = '-'.join([library_name, args.release, 'endf'])
+# the destination is decided after the release is known to avoid putting the release in a folder with a misleading name
+if args.destination == None:
```

Comment:
The Pythonic way to write this is:
```Python
if args.destination is None:
```

Reply:
now fixed, also fixed in convert_tendl and convert fendl